router/pump.go

```diff
@@ -361,6 +375,9 @@ func newContainerPump(container *docker.Container, stdout, stderr io.Reader) *co
 		logstreams: make(map[chan *Message]*Route),
 	}
 	pump := func(source string, input io.Reader) {
+		rateLimit := rateLimit()
+		burstLimit := int(rateLimit) * 2
+		limiter := rate.NewLimiter(rateLimit, burstLimit)
```

Comment:
Whoa golang has everything

Reply:
Right? ❤️

Reply:
Not in the stdlib, but yeah it's a decent library from the golang org.